packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -63,13 +64,13 @@ function SharePostContent({
         className="mb-5 mt-8"
       >
         <div className="flex max-w-full flex-col p-4 pt-5 laptop:flex-row">
-          <div className="mb-5 flex flex-1 flex-col laptop:mb-0">
+          <div className="mb-5 flex max-w-full flex-1 flex-col truncate laptop:mb-0">
```

Comment:
Need `truncate` when `TruncateText` already has it?

Reply:
Yeah, without `truncate`, it looks like this.
![image](https://github.com/user-attachments/assets/b3c30b13-5ea6-4f9d-bb5a-981e4fed0c66)

The title does get truncated, but still remain full width

Reply:
Technically it just needs `overflow: hidden` to fix its width, but I like using that `truncate` class at it adds a bit extra "protection" and is clear as why it is doing it
```css
.overflow-ellipsis, .text-ellipsis, .truncate {
    text-overflow: ellipsis;
}
.truncate {
    overflow: hidden;
    white-space: nowrap;
}
```